Podfile

```diff
@@ -0,0 +1,25 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '10.0'
+install! 'cocoapods', :disable_input_output_paths => true
+
+
+target 'Unleash' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for Unleash
+  pod "PromiseKit", "~> 6.8"
```

Comment:
Do we need to have this Podfile? Can we tell the `podspec` that a dependency this project has is `PromiseKit`?

Reply:
Its for the sample app

Reply:
But we can remove it

Reply:
Actually I'll remove it & update the readme